docs/_sections/file_care.md

```diff
@@ -3,9 +3,42 @@ layout: post
 title: File Care
 ---
 
-## Best Practices
+After transferring, analyzing, and (optionally) packaging DV streams, they will begin their new life as files. 
 
-## Options
+## Community-established practices
 
-## Platforms
+Digital preservation is an ever-evolving field, and decisions around good digital stewardship will differ between institutions, their collecting policies, their financial situations, and a myriad of other factors. Long-term preservation of hundreds of hours of DV camera footage for a small non-profit, DV documenting human rights concerns intended for use in a court case, and DV that are part of an artwork held by a national art museum will all require different levels, strategies, and approaches to preservation. While DVRescue provides the highest level of care to the transfer and analysis process, the longer term preservation actions and storage options will vary. All of this to say: the best digital preservation practices will be determined by you and your organization's preservation policies, and will likely be different from other preservation situations.
+
+In many ways, DV can be treated like any other bitstream data destined for preservation. If your institution already has established practices around handling digital objects for preservation and access, DV should fit within those frameworks as much as possible.
+
+If getting started with digital preservation for the first time, it is worth considering the [NDSA Levels of Digital Preservation](https://ndsa.org/publications/levels-of-digital-preservation/) and the [Digital Preservation Coalition's Digital Preservation Handbook](https://www.dpconline.org/handbook). These resources can get you started with some of the foundational elements of long-term file care, such as integrity checking, metadata guidelines, and storage requirements.
+
+## Creating archival access derivatives
+
+If you are currently using a digital preservation or archival access system that will create access derivatives for you, such as files suitable for giving to clients/patrons or for playback on the web... great! Allow that system to do the work for you.
+
+There are situations when will need to create your own copies of access files.
+
+MIPoPS uses and recommends the following settings:  
+
+- Picture Settings
+	- Size
+		- Anamorphic = Strict
+		- Cropping = Custom (if Automatic doesn’t crop out the head switching noise or black bars)
```

Comment:
huh, DV should haven't head switching noise or a black boundary.

Reply:
Right, unless the footage was transferred from an analog format.

Reply:
I was going to leave cropping in since I just digitized a TON of DVCPro tapes with head tracking noise because they were originally shot on an analog format and transferred to DV for some reason. But it seems to be fairly commonplace. However, maybe that's just been my experience? Thoughts?

Reply:
I'd lean towards not having cropping included myself - we have a ton of analog transfers to DV in our collection, but we also have a ton of camera originals which I wouldn't want cropped. Since in this case decision to crop would be content based, rather than format based like analog (always has head switching noise inherently) if I had to pick I'd prioritize not cropping due to it not being an inherent issue with DV. (If that makes sense)

Reply:
Right, yes, that does make sense. That's what I meant, like leaving it as "Custom" makes it so that you can do it on a case by case basis. The way I have it set it so that you can crop as needed. Like the preset will open the cropping at 0x0x0x0 and you can adjust accordingly in Handbrake.

Reply:
ah - sweet! yeah, I haven't done a JSON import to Handbrake for settings - if 0x0x0x0 is default then that makes sense to me!

Reply:
Sweet, thank you @privatezero !!!